addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview_tests.js

```diff
@@ -63,8 +63,8 @@ QUnit.test('mark as read', async function (assert) {
         hasChatWindow: true,
         hasMessagingMenu: true,
         async mockRPC(route, args) {
-            if (route.includes('set_message_done')) {
-                assert.step('set_message_done');
+            if (route.includes('mark_all_as_read')) {
+                assert.step('mark_all_as_read');
```

Comment:
Don't like having this technical detail in test. Would remove this part (+ related assertion(s)) from this test.
If you feel the need to cover this, make another technical test that is specific to assert `mark_all_as_read` RPC.